java/client/src/main/java/glide/api/models/commands/scan/HScanOptions.java

```diff
@@ -10,4 +15,51 @@
  * @see <a href="https://valkey.io/commands/hscan/">valkey.io</a>
  */
 @SuperBuilder
-public class HScanOptions extends BaseScanOptions {}
+public class HScanOptions extends BaseScanOptions {
+
+    /** Option string to include in the HSCAN command when values are not included. */
+    public static final String NO_VALUES_API = "NOVALUES";
+
+    /**
+     * When set to true, the command will not include values in the results. This option is available
+     * from Redis version 8.0.0 and above.
+     */
+    @Builder.Default protected boolean noValues = false;
+
```

Comment:
Add this
```java
    public static class HScanOptionsBuilder {

        /** copy doc here. */
        public HScanOptionsBuilder noValues() {
            return noValues(true);
        }
    }
```

Reply:
Add same for zscan options

Reply:
Actually, you can do `noValuesOptions` (or so) class, which would be inherited by hscan opts and zscan opts, so you can avoid code duplicating